test/unit/seed/seeder.js

```diff
@@ -67,3 +67,33 @@ describe('Seeder.loadExtensions', function() {
       });
   });
 });
+
+describe('Seeder._waterfallBatch', function() {
+  var config = {
+    client: 'postgres',
+    connection: {
+      user: 'postgres',
+      password: '',
+      host: '127.0.0.1',
+      database: 'knex_test',
+    },
+    seeds: {
+      directory: 'test/unit/seed/test',
+    },
+  };
+  var seeder;
+
+  beforeEach(function() {
+    seeder = knex(config).seed;
+  });
+
+  it('should throw an error with correct file name', () => {
+    return seeder
+      ._waterfallBatch(['1-first.js', '2-second.js'])
+      .catch((error) => {
+        expect(error.message).to.match(
```

Comment:
This could use done() callback call, otherwise test will pass even if no errors are thrown.

Reply:
Good catch, fixing right away.
(I've added missing test files too.)

Reply:
Done, resolving conversation.